src/test/functional/steps/court-addresses.ts

```diff
@@ -81,6 +81,11 @@ Then('I enter {string} in the postcode textbox', async (postcode: string) => {
   await populateField(selector, postcode);
 });
 
+Then('I enter {string} in the primary epim id textbox', async (badepim: string) => {
+  const selector = '#primaryAddressEpimId';
```

Comment:
I like the idea of having a descriptive method for entering some text, but at the same time I think it could be a resuable one

Might be worth creating something like: 

Then('I enter {string} in the selected textbox', async (selector: string, value: string) => {
    await populateField(selector, value);
});

Reply:
changed